Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -391,4 +410,4 @@ public struct InAppMessageViewRepresentable: UIViewRepresentable {
         }
     }
 }
-#endif
+#endif // swiftlint:disable:this file_length
```

Comment:
I would be OK moving the SwiftUI code into a separate file to resolve this swift lint error. Also OK with keeping all of it in 1 file.

Reply:
IMO, it will be a good idea to move SwiftUI code into a separate file.

Reply:
[Opened a new PR](https://github.com/customerio/customerio-ios/pull/781) with that change.